### PR TITLE
Adding builder option to pif.load[s]

### PR DESCRIPTION
**Pull request by maxhutch**
This way, it can deserialize non-top-level objects in python

### CODE REVIEW

File: pypif/pif.py [hunk 1]
```diff
@@ -26,51 +26,53 @@ def dumps(pif, **kwargs):
     return json.dumps(pif, cls=PifEncoder, **kwargs)
 
 
-def load(fp, **kwargs):
+def load(fp, builder=None, **kwargs):
```
Comment (kjaym) [inline pypif/pif.py]: Is builder a common name for this?

File: pypif/pif.py [hunk 2]
```diff
@@ -26,51 +26,53 @@ def dumps(pif, **kwargs):
...
     :param fp: File-like object supporting .read() method to deserialize from.
     :param kwargs: Any options available to json.load().
     :return: Single object derived from :class:`.Pio` or a list of such object.
     """
-    return loado(json.load(fp, **kwargs))
+    return loado(json.load(fp, **kwargs), builder=builder)
 
 
-def loads(s, **kwargs):
+def loads(s, builder=None, **kwargs):
```
Comment (kjaym) [inline pypif/pif.py]: Can you add to the comments for each method and give the type of builder as well as its purpose.

File: pypif/pif.py (hunk 1 above)
Comment (maxhutch) [reply pypif/pif.py]: I don't think so.  `class`, `type`, and `object` are python keywords.  Open to suggestions.

Comment (kjaym) [reply pypif/pif.py]: I see clazz used in java world very commonly.

Comment (maxhutch) [reply pypif/pif.py]: https://stackoverflow.com/questions/38718484/whats-convention-for-naming-a-class-or-method-as-class-in-python

**Pull request merged**